### DISCUSSION TITLE
Feature: Allow "external" internal Links

### DISCUSSION THREAD

**Discussion by kobykotiv** [IceWhaleTech/CasaOS#405] [Ideas] [answered]
**Is your feature request related to a problem? Please describe.**
I LOVE this web app to manage my cloud server. I have many services running but when an 'app' has the WEB UI link configured for e.g. `n8n.cooldomain.com` or `n8n.awesomesauce.duckdns.org` I can no longer just click to open it because that url is technically external. Internal IP address links still let me open the 'app' by just clicking on the icon. e.g. 192.168.10.7:3333

**Describe the solution you'd like**
I'd really like the CASAOS apps screen to allow 'external' internal links. I understand if everything needs to be secure by default. 

**Describe alternatives you've considered**
[Dashy](https://github.com/Lissy93/dashy) is a self-hosted start screen  bookmark manager that plays nice with reverse proxies! It'd be amazing if CasaOS caught up to Dashy before I switch back. Dashy as far as I know doesn't have an integrated docker-compose deployment hook that adds an icon for each installed or running service.

**Comment by JohnGuan** [comment, marked as answer]
Thanks for the idea, which has been implemented in [v0.3.6](https://github.com/IceWhaleTech/CasaOS/releases/tag/v0.3.6)!
